### PR TITLE
cargo-sweep 0.7.0 (new formula)

### PR DESCRIPTION
**Pull request by fahasch**
Utility for cleaning up unused build files generated by Cargo

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by ZhongRuoyu — DISMISSED** [review]

File: Formula/c/cargo-sweep.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  head "https://github.com/holmgr/cargo-sweep.git", branch: "master"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--no-default-features", *std_cargo_args
+  end
+
+  test do
+    assert_equal "cargo-sweep #{version}", shell_output(bin/"cargo-sweep -V").strip
```
Comment (chenrui333) [inline Formula/c/cargo-sweep.rb:16]: can you check out the other plugin and write a more substantial test with `rust-init`?

Comment (fahasch) [reply Formula/c/cargo-sweep.rb:16]: I was thinking about it. However, it is not so  clear what would be a good test. The utility deletes unused Cargo build files. For that: 1) cargo has to be used to compile something. 2) some days have to pass 3) then the build resources will be deleted.

To model this, one would need to set the time-stamps of some of the build files. I am afraid however that such a procedure will be very hard to maintain as the build files of cargo as well as the setup of the build directories might change in the future.

**Review by chenrui333 — APPROVED** [review]
lgtm, thanks!

**Review by ZhongRuoyu — APPROVED** [review]
Thanks!

**Pull request merged**